app/views/catalog/_home_text.html.erb

```diff
@@ -33,12 +33,4 @@
 <div id="about" class="card">
   <h2 class='card-header collapsed collapse-toggle' data-toggle="collapse" data-target="#about-content"><a href="/rails/info/properties">About your application&rsquo;s environment</a></h2>
   <div id="about-content" class="card-body collapse"></div>
-</div>
-
-<script>
-  Blacklight.onLoad(function() {
```

Comment:
I'm curious about why this needed to be removed.

Reply:
jQuery wasn't loaded yet: https://git.psu.edu/i-tech/psulib_blacklight/issues/75. This was only a problem on the homepage.